### PR TITLE
major overhaul to main README.md

### PR DESCRIPTION
**Pull request by cuttlefishh**
I have made significant effort to clean up and update the front page of the EMP GitHub repo. This is important as the manuscript specifically points users here: "Instructions and code for using Redbiom with the EMP catalogue are available at github.com/biocore/emp." Please review when you have time. Then let's merge these changes ASAP as it's possible referees may be going here to learn more. For sure when the paper is published we want everything to be clear and straightforward because we are making a big deal about the utility of the resource.

### CODE REVIEW

**Comment by cuttlefishh** [comment]
Would especially value feedback from @wasade @antgonza @mortonjt @tanaes. Thanks guys!

File: README.md [hunk 1]
```diff
@@ -3,17 +3,67 @@ Earth Microbiome Project
...
+* **Contribute samples.** We are not currently soliciting samples for the EMP. If you have an idea for samples you might like to submit in the future, you may [email](mailto:lukethompson@gmail.com) the project leader for the EMP, Dr. Luke Thompson.
+
+Using the EMP catalogue
+-----------------------
+
+The EMP catalogue is a large, diverse, standardized dataset of microbiomes for use by the public. Here are some of the ways you can use this resource:
+
+* **Download individual studies from the Qiita EMP Portal.** [Qiita](https://qiita.ucsd.edu/emp/) requires the Google Chrome browser. For each study, you can download metadata (mapping file), feature tables (BIOM file), and demultiplexed raw sequence files.
+* **Download EMP Release 1 from our FTP site.** [EMP Release 1](ftp://ftp.microbio.me/emp/release1) contains merged and quality-filtered mapping files, BIOM tables, OTU/sequence information, and alpha/beta-diversity results for the 97 studies in the initial meta-analysis of the EMP. The FTP site contains README files about its contents, and the individual files are listed [here](https://github.com/biocore/emp/blob/master/data/data_locations.txt).
```
Comment (tkosciol) [inline README.md]: I think this section should go first.

File: README.md [hunk 2]
```diff
@@ -3,17 +3,67 @@ Earth Microbiome Project
...
+* **Merge your data with all or part of the EMP dataset.** If you sequenced your sample using the [EMP 16S rRNA primers](http://www.earthmicrobiome.org/protocols-and-standards/16s/) and picked OTUs using either Deblur or closed-reference against Greengenes 13.8 or Silva 123, you can merge your BIOM table with the relevant merged EMP Release 1 BIOM table or one of the individual per-study BIOM tables from Qiita. Basic instructions for processing your data are [here](http://www.earthmicrobiome.org/protocols-and-standards/initial-qiime-processing/). You can then use [QIIME1](http://qiime.org/) or [QIIME2](https://qiime2.org/) to merge the BIOM tables and mapping files.
```
Comment (tkosciol) [inline README.md]: add link to deblur manuscript?

File: README.md (hunk 1 above)
Comment (cuttlefishh) [inline README.md]: Agree. Done!

File: README.md (hunk 2 above)
Comment (cuttlefishh) [inline README.md]: Done. Thanks!

**Pull request merged**